toolchain/check/import_ref.cpp

```diff
@@ -2170,17 +2181,16 @@ static auto AddInterfaceDefinition(ImportContext& context,
                                    const SemIR::Interface& import_interface,
                                    SemIR::Interface& new_interface,
                                    SemIR::InstId self_param_id) -> void {
-  new_interface.scope_id = context.local_name_scopes().Add(
-      new_interface.first_owning_decl_id, SemIR::NameId::None,
-      new_interface.parent_scope_id);
   auto& new_scope = context.local_name_scopes().Get(new_interface.scope_id);
-  new_scope.set_is_interface_definition();
   const auto& import_scope =
       context.import_name_scopes().Get(import_interface.scope_id);
 
   // Push a block so that we can add scoped instructions to it.
   context.local_context().inst_block_stack().Push();
-  AddNameScopeImportRefs(context, import_scope, new_scope);
+  new_scope.set_is_interface_definition();
+  InitializeNameScopeAndImportRefs(
```

Comment:
Can we reverse the order of these calls? Setting properties on the scope before we call `InitializeNameScope` looks a little surprising to me.